library/options.inc.php

```diff
@@ -716,25 +895,20 @@ function generate_form_field($frow, $currvalue)
                     if ($codestring === '') {
                         continue;
                     }
-
-                    $code_text = lookup_code_descriptions($codestring);
                     if ($currdescstring !== '') {
                         $currdescstring .= '; ';
                     }
-
-                    if (!empty($code_text)) {
-                        $currdescstring .= $code_text;
-                    } else {
-                        $currdescstring .= $codestring;
-                    }
+                    $currdescstring .= getCodeDescription($codestring, $codetype);
```

Comment:
i don't think this function is looking up codes correctly (ie. just looking at codes table instead of what lookup_code_descriptions() does which is supports both internal and external code types

Reply:
OK I'm changing getCodeDescription() to call lookup_code_descriptions(). Thanks.